recipes/lfpy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = 'LFPy' %}
+{% set version = "2.0.2" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/l/lfpy/{{name}}-{{ version }}.tar.gz
+  sha256: ab5103f6fd5fc9fb4005e65033a98d9d67f9ace4bc4b9be6c3462eab65539698
+
+build:
+  number: {{ build }}
+  script: "{{PYTHON}} -m pip install . --no-deps -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - cython >=0.20
+    - numpy >=1.8
+    - neuron
+  run:
+    - python
+    - setuptools
+    - cython >=0.20
+    - numpy >=1.8
```

Comment:
In the ``run`` dependency, I think, it should be
```
- {{ pin_compatible('numpy') }}
```
for ``numpy`` as you are building against it